Added DAO Engineer JD

idiom-bytes opened the pull request:
Opening up PR for DAO Engineer JD

idiom-bytes committed:
commiting first pass at dao engineer JD.

idiom-bytes committed:
Updating README to link to DAO Engineer.

idiom-bytes committed:
Update dao.md

idiom-bytes committed:
Update dao.md

idiom-bytes committed:
Update dao.md

idiom-bytes committed:
Update dao.md

LoznianuAnamaria commented on dao.md:
```diff
@@ -0,0 +1,40 @@
+# 🦞 Senior Blockchain Developer - OceanDAO
+[Ocean Protocol](https://www.oceanprotocol.com/) has a vibrant, expanding ecosystem of engineers, data scientists, neuroscientists, ambassadors, podcasters, that are build traction across 5 pillars of growth, receiving grants and funding to deliver results. To serve them better, we’re hiring!
+
+[OceanDAO](https://www.oceanprotocol.com/dao) is a key component that brings decentralization and scale to participants that are driving traction into the Ocean ecosystem. We’re growing the OceanDAO team. If you love to work with crypto, to build and to engage deeply with an enthusiastic community, we want to hear from you.
```
If you love to work with crypto, to build and **_to_** engage.... remove the 'to' in front of engage

LoznianuAnamaria commented on dao.md:
```diff
@@ -0,0 +1,40 @@
+
+## Minimum Qualifications
+* 6+ months of Solidity experience
+* 3+ years backend experience, including NodeJS
```
I would order the experience needed based on years, in reverse order. To 4+ years of software experience to be the first one, then 3+ years of backend the second one and the 6+ months, the last one

LoznianuAnamaria commented on dao.md:
```diff
@@ -0,0 +1,40 @@
+* Excellent writing and communication skills. Candidate will engage with the community, receive feedback, identify usage patterns, and communicate macro outcomes.
+* Successfully delivered on promises, comfortable working autonomously and within a team
+* Differentiate good and bad feedback to find real pain points to improve product/program based on learnings and manage and update scripts and reward function accordingly
+* Proven experience in building meaningful technical communities around projects and ability to build and manage relationship with a key stakeholders
```
* Manage relationships(plural)
* With key stakeholders, remove the 'a' key stakeholders

LoznianuAnamaria commented on dao.md:
```diff
@@ -0,0 +1,40 @@
+* 2+ years working with cloud infrastructure, supporting devops.
+* 1+ years CI/CD experience (e.g. Travis, tox)
+* You’re obsessed with blockchain and DAOs. You play with dapps, vote on proposals, or have contributed code to a DAO.
+* Excellent writing and communication skills. Candidate will engage with the community, receive feedback, identify usage patterns, and communicate macro outcomes.
```
Instead of 'Candidate', I would make this personal. You will engage with the community...

idiom-bytes committed:
Update dao.md

Implementing Ana's recommendations. Thank you!

idiom-bytes commented:
Ok, I've updated the description based on your recommendations. Thank you!

@trentmc I have adjusted the technical expectations to focus more on web3/backend/infra. This reflects the intention on separating backend/frontend initiatives, in addition to decentralizing more frontend stuff (i.e. Uni Labs vs. Uni FE debacle). 

Moreover, Ocean whitepaper and looking out further into the future (beyond 2022) may bring more onchain systems, and perhaps even para-chain implementations of voting/funding/etc to make these systems low-cost & truly web3/decentralized (vs. off-chain gasless voting on proposals via snapshot.... which is same same, but different). 

Please LMK if this checks out with your intuitions, or if you want to put some emphasis on full-stack.

trentmc commented on dao.md:
```diff
@@ -0,0 +1,40 @@
+## Minimum Qualifications
+* 4+ years of software experience
+* 3+ years backend experience, including NodeJS
+* 6+ months of Solidity experience
```
Also 1+ years web3.js

trentmc commented on dao.md:
```diff
@@ -0,0 +1,40 @@
+## Your Next Step?
+If you love to build things and care deeply about the future of data, we want to hear from you! Please email [devjobs@oceanprotocol.com](mailto:devjobs@oceanprotocol.com), with:
+* your cover letter - why you’d like to join
+* your CV
```
Also ask for eth address(es) or ens names. Call it optional. This will be super useful for us though!

trentmc commented:
Overall I'm fine with it.
